tests/digraph/test_pred_succ.py

```diff
@@ -18,85 +18,123 @@
+        self.assertEqual(["c"], res)
 
     def test_many_children(self):
         dag = retworkx.PyDAG()
-        node_a = dag.add_node('a')
+        node_a = dag.add_node("a")
         for i in range(10):
-            dag.add_child(node_a, {'numeral': i}, {'edge': i})
+            dag.add_child(node_a, {"numeral": i}, {"edge": i})
         res = dag.successors(node_a)
-        self.assertEqual([{'numeral': 9}, {'numeral': 8}, {'numeral': 7},
-                          {'numeral': 6}, {'numeral': 5}, {'numeral': 4},
-                          {'numeral': 3}, {'numeral': 2}, {'numeral': 1},
-                          {'numeral': 0}], res)
+        self.assertEqual(
+            [
+                {"numeral": 9},
+                {"numeral": 8},
+                {"numeral": 7},
+                {"numeral": 6},
+                {"numeral": 5},
+                {"numeral": 4},
+                {"numeral": 3},
+                {"numeral": 2},
+                {"numeral": 1},
+                {"numeral": 0},
+            ],
+            res,
+        )
 
 
 class TestBfsSuccessors(unittest.TestCase):
     def test_single_successor(self):
         dag = retworkx.PyDAG()
-        node_a = dag.add_node('a')
-        node_b = dag.add_child(node_a, 'b', {'a': 1})
-        node_c = dag.add_child(node_b, 'c', {'a': 2})
-        dag.add_child(node_c, 'd', {'a': 1})
+        node_a = dag.add_node("a")
+        node_b = dag.add_child(node_a, "b", {"a": 1})
+        node_c = dag.add_child(node_b, "c", {"a": 2})
+        dag.add_child(node_c, "d", {"a": 1})
         res = retworkx.bfs_successors(dag, node_b)
-        self.assertEqual([('b', ['c']), ('c', ['d'])], res)
+        self.assertEqual([("b", ["c"]), ("c", ["d"])], res)
 
     def test_many_children(self):
         dag = retworkx.PyDAG()
-        node_a = dag.add_node('a')
+        node_a = dag.add_node("a")
         for i in range(10):
-            dag.add_child(node_a, {'numeral': i}, {'edge': i})
+            dag.add_child(node_a, {"numeral": i}, {"edge": i})
         res = retworkx.bfs_successors(dag, node_a)
-        self.assertEqual([('a', [{'numeral': 9}, {'numeral': 8},
-                          {'numeral': 7}, {'numeral': 6}, {'numeral': 5},
-                          {'numeral': 4}, {'numeral': 3}, {'numeral': 2},
-                          {'numeral': 1}, {'numeral': 0}])], res)
+        self.assertEqual(
+            [
+                (
+                    "a",
+                    [
+                        {"numeral": 9},
+                        {"numeral": 8},
+                        {"numeral": 7},
+                        {"numeral": 6},
+                        {"numeral": 5},
+                        {"numeral": 4},
+                        {"numeral": 3},
+                        {"numeral": 2},
+                        {"numeral": 1},
+                        {"numeral": 0},
+                    ],
+                )
+            ],
+            res,
+        )
```

Comment:
Wow this is pretty ugly.

Reply:
Perhaps the beauty is that we didn't have to format it ourselves!